src/main/webapp/open_hours.js

```diff
@@ -0,0 +1,99 @@
+document.addEventListener("DOMContentLoaded", async function() {
+  if(document.getElementById("registration-form-area")) {
+  const mondayTimeOption = new TimeOption("Monday", true, null);
+  const tuesdayTimeOption = new TimeOption("Tuesday", true, null);
+  const wednesdayTimeOption = new TimeOption("Wednesday", true, null);
+  const thursdayTimeOption = new TimeOption("Thursday", true, null);
+  const fridayTimeOption = new TimeOption("Friday", true, null);
+  const saturdayTimeOption = new TimeOption("Saturday", true, null);
+  const sundayTimeOption = new TimeOption("Sunday", true, null);
+  }
+});
+
+class TimeOption {
+  constructor(day, forRegistration, organization) {
+    this.day = day;
+    this.forRegistration = forRegistration;
+    this.organization = organization;
+    this.optionArea = document.getElementById("hours-option-area");
+    if(organization != null) {
+      if(day == "Monday") {
+        this.organizationIsOpen = organization.mondayIsOpen;
```

Comment:
I think it could be confusing to have organizationIsOpen when it's holding open/close status for just one day!

Reply:
I understand what you mean. These objects are constructed for unique days so it is trying to refer to if the organization is open for the day it was created for but I was thinking I might need this for when I use it for the edit page but I think i figured out another way so i removed it!